tools/linter_lib/custom_check.py

```diff
@@ -889,7 +889,10 @@
         {
             "pattern": r"\b[rR]ealm[s]?\b",
             "include_only": {"templates/zerver/help/"},
-            "exclude": {"templates/zerver/help/change-organization-url.md"},
+            "exclude": {
+                "templates/zerver/help/change-organization-url.md",
+                "templates/zerver/help/include/rest-endpoints.md",
+            },
```

Comment:
This would result in someone using `realm` for the sidebar of the api docs, any other solution to use `/api/update-realm-user-settings-defaults` without getting error.

Reply:
We probably should exclude `-realm-` with `exclude_pattern` as a cleaner solution for this.

Reply:
Done